### PR TITLE
Add local Goodwill affiliates as operators

### PR DESCRIPTION
**Pull request by 1ec5**
Added a number of local [Goodwill](https://www.wikidata.org/wiki/Q5583655) affiliates to the `operator` tree. These are the Goodwill affiliates that have been tagged (correctly or incorrectly) in OpenStreetMap so far, plus any other affiliates that Wikidata happened to have an item for. This list is likely missing some of the largest affiliates in the United States, but I couldn't readily find a list of them to guide this PR.

## Rationale

The Goodwill brand is shared among a network of independent nonprofits, many of which retain some degree of branding. For example, Goodwill of Silicon Valley posts its full name prominently at each of its stores and donation centers, as well as on its trucks and donation bins. On the other hand, some other Goodwill affiliates use the parent organization’s branding as is.

[kiely](https://www.mapillary.com/app/?pKey=278303837272609)

People tend to care which organization is enabling them to get a tax deduction on their donated items. The Wikidata items are tagged with IRS EINs and Charity Navigator profile IDs, so there could be a real use case for cross-referencing the operator.

I think it makes sense to keep all the stores tagged with the same `brand` for the parent organization but with different `operator`s. This differs from how we’re dealing with Caterpillar in #5647, but anyways there’s a significant difference between a dealer network and how Goodwill operates.

## Caveats

* I’m not sure if the new entries should specify `shop=charity`, since that would be covered by Goodwill’s existing `brand` entry. Can I omit it, in case someone tags a donation bin with `amenity=recycling` and `operator=Goodwill of …`?
* I set the `locationSet` of each affiliate to the state or states they operate in. For Wikidata, I did a bit more research and came up with the specific counties that some of them operate in, but I didn’t bother to add any GeoJSONs for those service areas to NSI.
* Some of the Wikidata items are tagged with TikTok and Snapchat user names, in case anyone feels like adding [social links](https://github.com/osmlab/name-suggestion-index/blob/fce09fc75976493749ef2162b666211c3bc3ff3d/app/src/CategoryRowSocialLinks.jsx#L17-L64) for them. I didn’t, because I still don’t get it.

### CODE REVIEW

**Comment by arch0345** [comment]
This situation is similar to GoHealth Urgent Care (#5648): local companies seem to operate the same brand

**Comment by bhousel** [comment]
> * I’m not sure if the new entries should specify `shop=charity`, since that would be covered by Goodwill’s existing `brand` entry. Can I omit it, in case someone tags a donation bin with `amenity=recycling` and `operator=Goodwill of …`?

I think all of these should just be in the brand file?  

e.g. an item should look like:
```js
    {
      "displayName": "Goodwill (Florida)",
      "locationSet": {
        "include": ["us-fl.geojson"]
      },
      "tags": {
        "brand": "Goodwill",
        "brand:wikidata": "Q5583655",
        "brand:wikipedia": "en:Goodwill Industries",
        "name": "Goodwill",
        "operator": "Goodwill Industries of South Florida",
        "operator:wikidata": "Q54624094",
        "shop": "charity"
      }
    },
```

That's like saying, Goodwill (Florida) will override the Goodwill (USA), but only in Florida.

It's better to think of these as presets that are exclusively applied in a particular region - not as rulesets that can combine together depending on what tags are present.

**Comment by 1ec5** [comment]
Interesting, I didn’t realize brand entries could specify `operator`s like that. I think the display names would still have to be more specific, because I didn’t create features for each affiliate. Some affiliates divide up counties, while others have overlapping service areas.

File: data/operators/shop/charity.json [hunk 1]
```diff
@@ -4,6 +4,18 @@
     "exclude": {"generic": ["^charity$"]}
   },
   "items": [
+    {
+      "displayName": "Goodwill Industries of the Columbia Willamette",
+      "id": "goodwillindustriesofnorthlouisiana-239b23",
```
Comment (1ec5) [inline data/operators/shop/charity.json:9]: Needs a different ID.

**Comment by arch0345** [comment]
Merging this as is, will continue to add more affiliates in the future

**Pull request merged**